Wangscape/noise/module/CornerCombinerBase.h

```diff
@@ -4,6 +4,8 @@ namespace noise
 {
 namespace module
 {
+// Default power for the noise::module::CornerCombinerBase module.
```

Comment:
Hmm... ok. In the example it looks like annotations of some documentation generation tool :thinking:.